app/authzed/api/_meta.ts

```diff
@@ -1,15 +1,15 @@
+import type { MetaRecord } from "nextra";
```

Comment:
Nextra fails cryptically when you have a `_meta.ts` file with the wrong shape. Adding this type assertion helps prevent that.